### DISCUSSION TITLE
The error about “Isaac-Lift-Franka-v0” task

### DISCUSSION THREAD

**Discussion by Aqua12138** [isaac-sim/IsaacLab#26] [Q&A] [answered]
Why is the cube not responding physically when I run lift env, I tried controlling the franka to clip it but there doesn't seem to be any collision response
![image](https://user-images.githubusercontent.com/88502156/222345467-f2fd55e8-3587-4e53-9cb8-7896e1b686ea.png)
![image](https://user-images.githubusercontent.com/88502156/222345549-318ac33e-a313-435b-8dd5-0152f9ac94c6.png)

**Comment by Mayankm96** [comment, marked as answer]
Hi, thanks a lot for reporting this!

The main issue here was that the cube was also being used as a marker in this example. However, to make it a marker, the marker class removed all physics APIs from the cube. This also affected the asset reference being used for making the cube for manipulation. I have added a note about this in the "Known Issues" section of the documentation. 

This should be fixed now in PR #30.
